### PR TITLE
Update Actions.invoke to return the result of invoking the action.

### PR DESCRIPTION
**Pull request by tvolkert**
## Description

`Actions.invoke()` was returning a boolean value corresponding to whether the dispatcher's `invokeAction()` method returned a non-null value.  True meant that the action was enabled and itself returned a non-null value, and false indicated that either the action was disabled, or it returned a null value from its `invoke()` method.

This boolean value is less useful than just returning the result of invoking the action itself. In my (admittedly limited) experience in playing with actions so far, the caller is likely going to want to just get the value that the action returned when it was invoked.

## Tests

I added a test for the return value and updated a test that was relying on getting "false" back for a null value.

## Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I signed the [CLA].
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] All existing and new tests are passing.
- [x] The analyzer (`flutter analyze --flutter-repo`) does not report any problems on my PR.
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

- [x] No, no existing tests failed, so this is *not* a breaking change.
- [ ] Yes, this is a breaking change. *If not, delete the remainder of this section.*

### CODE REVIEW

**Comment by gspencergoog** [comment]
Yes, I've thought about this before, and gone back and forth, but if it seems generally useful, then I'm fine with this.  I didn't do it originally because I was trying to minimize the API breakage, and because if we do what you have in this PR, then there isn't an easy way to find out if the reason it returns null is because it couldn't find it or it was disabled, or if it just returned null.

It is possible to check ahead of time that an action is available now, via `Actions.find`, so that's less of an issue.

**Review by gspencergoog — APPROVED** [review]
32384589-a60f0e74-c078-11e7-9bc1-e5b5287aea9d

File: packages/flutter/lib/src/widgets/actions.dart [hunk 1]
```diff
@@ -582,6 +582,10 @@ class Actions extends StatefulWidget {
   ///
   /// Setting `nullOk` to true means that if no ambient [Actions] widget is
   /// found, then this method will return false instead of throwing.
```
[the remark this answers is not in the document]
Comment (tvolkert) [reply packages/flutter/lib/src/widgets/actions.dart]: Done

File: packages/flutter/lib/src/widgets/actions.dart [hunk 2]
```diff
@@ -582,6 +582,10 @@ class Actions extends StatefulWidget {
...
+  ///
+  /// Returns the result of invoking the action's [Action.invoke] method. If
+  /// no action mapping was found for the specified intent, or if the action
+  /// that was found was disabled, then this returns null.
```
[the remark this answers is not in the document]
Comment (tvolkert) [reply packages/flutter/lib/src/widgets/actions.dart]: Done

**Pull request merged**